packages/react-server-webpack-plugin/src/react-server-webpack-plugin.ts

```diff
@@ -110,10 +116,15 @@ function clientSource() {
     ${HEADER}
     import React from 'react';
     import ReactDOM from 'react-dom';
-    import {showPage} from '@shopify/react-html';
-    import App from 'index';
+    import {showPage, getSerialized} from '@shopify/react-html';
+
+    import App from './index';
```

Comment:
this line is failing for me as it cannot find `./index`

Reply:
🤔  I thought I had left this as 'index' instead of '/index'm if you change it back does it work?

Reply:
yup changing it back to `index` works fine.

Reply:
Yah, I recall that the webpack plugin needed `index` instead of `./index`. It was a weird thing I explored/debugged ~8-10 months ago when we first started the plugin.